mobile/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { TouchableOpacity } from '#libs/components/components';
+
+import { styles } from './styles';
+
+type IconButtonProperties = {
```

Comment:
```suggestion
type Properties = {
```

Reply:
i dont think we should strictly name them only Properties 🤔

Reply:
@iamAlinaaa, but we consistently name the props 'Properties' everywhere. I think you should rename it